db/migrate/20180619122743_add_rating_to_recipes.rb

```diff
@@ -0,0 +1,5 @@
+class AddRatingToRecipes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :recipes, :rating, :integer
```

Comment:
I would rename this column to `average_rating` if you want to store this value in the db. It can be a dynamic attribute as well. If you choose that, you can remove this migration altogether.

Reply:
Column is renamed to `average_rating`